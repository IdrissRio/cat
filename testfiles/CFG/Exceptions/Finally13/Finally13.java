public class Finally13 {
  void m10() throws TestException {
    try {
      int a;
      throw new TestException();
    } catch (TestSubException e) {
      int b;
    } finally {
      int c;
    }
    int d;
  }
}

class TestException extends Exception {}

class TestSubException extends TestException {}