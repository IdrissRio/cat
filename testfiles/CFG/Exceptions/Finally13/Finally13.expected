NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [VariableDeclarator[47]]
NodeName: VariableDeclarator[47] Successors: [ThrowStmt[50]]
NodeName: ThrowStmt[50] Successors: [ClassInstanceExpr[52]]
NodeName: ClassInstanceExpr[52] Successors: [UncheckedExceptions[51]]
NodeName: UncheckedExceptions[51] Successors: [VariableDeclarator[68]]
NodeName: VariableDeclarator[68] Successors: [Exit[37]]
NodeName: Exit[37] Successors: []
NodeName: Entry[118] Successors: [Exit[119]]
NodeName: Exit[119] Successors: []
NodeName: Entry[110] Successors: [Exit[111]]
NodeName: Exit[111] Successors: []
NodeName: Entry[141] Successors: [Exit[142]]
NodeName: Exit[142] Successors: []
NodeName: Entry[133] Successors: [Exit[134]]
NodeName: Exit[134] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[37] Predecessor: [VariableDeclarator[68]]
NodeName: VariableDeclarator[68] Predecessor: [UncheckedExceptions[51]]
NodeName: UncheckedExceptions[51] Predecessor: [ClassInstanceExpr[52]]
NodeName: ClassInstanceExpr[52] Predecessor: [ThrowStmt[50]]
NodeName: ThrowStmt[50] Predecessor: [VariableDeclarator[47]]
NodeName: VariableDeclarator[47] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: Exit[119] Predecessor: [Entry[118]]
NodeName: Entry[118] Predecessor: []
NodeName: Exit[111] Predecessor: [Entry[110]]
NodeName: Entry[110] Predecessor: []
NodeName: Exit[142] Predecessor: [Entry[141]]
NodeName: Entry[141] Predecessor: []
NodeName: Exit[134] Predecessor: [Entry[133]]
NodeName: Entry[133] Predecessor: []
